ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -111,25 +105,36 @@ open class PillButton: UIButton, FluentUIWindowProvider {
         }
     }
 
-    public override func layoutSubviews() {
-        super.layoutSubviews()
-        updateUnreadDot()
+    @objc public static let cornerRadius: CGFloat = 16.0
+
+    @objc public let pillBarItem: PillButtonBarItem
+
+    @objc public let style: PillButtonStyle
```

Comment:
I think you'll need to call `updatePillButtonTokens()` when this is set as well?

Reply:
Even though it's a let? I don't think you can do didSet on a let. Should it change to a var? I don't think this is meant to change after initialization.

Reply:
Ah good catch, totally missed that it's a `let`. Okay, if this can't change then it doesn't matter!